modules/logging-loki-gui-install.adoc

```diff
@@ -14,43 +14,39 @@ To install and configure logging on your {product-title} cluster, additional Ope
 
 .Procedure
 
-. Install the `Loki Operator`:
+. To install the `Loki Operator` in the {product-title} web console, click *Operators* -> *OperatorHub*.
 
-.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+. Type `Loki Operator` in the filter by keyword box. Choose *Loki Operator* from the list of available Operators and click *Install*.
 
-.. Type `Loki Operator` in the filter by keyword box. Choose *Loki Operator* from the list of available Operators and click *Install*.
-+
 [NOTE]
 ====
 The Community Loki Operator is not supported by Red Hat.
 ====
 
-.. On the Install Operator page, for *Update Channel* select *stable*.
+. On the Install Operator page, for *Update Channel* select *stable*.
```

Comment:
Pretty sure this is "**Update channel**" now, please check the capitalization in the console. I think I had a Jira opened recently to fix this in a different module.
```suggestion
. On the *Install Operator* page, for *Update channel* select *stable*.
```